experimental/iterators/test/Integration/Dialect/Iterators/CPU/print.mlir

```diff
@@ -2,11 +2,19 @@
 // RUN: mlir-cpu-runner -e main -entry-point-result=void \
 // RUN: | FileCheck %s
 
+func.func @print_empty_tuple(%tuple : tuple<>) -> () {
+  "iterators.printtuple"(%tuple) : (tuple<>) -> ()
```

Comment:
note: printtuple reads quite poorly -> "print_tuple".
Also, print is one of those ops that is probably nicer to have as polymorphic.

Reply:
Fair point. Changing that wouldn't be part of this PR, though, right?

Reply:
that's def a separate PR